doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md

```diff
@@ -0,0 +1,176 @@
+
+### A Note on disabling Kernel Journaling
+
+Disabling journaling on an EXT4 filesystem offers a few tangible benefits, but they come with significant trade-offs, especially in terms of data integrity and stability. Below are the pros and cons of disabling journaling.
+
+#### **Advantages of Disabling Journaling**
+
+1. **Reduced Write Overhead**: Disabling journaling reduces write overhead, as the system doesn’t need to log changes twice (once in the journal, then in the actual data). This could improve write performance, especially for I/O-intensive applications.
+
+2. **Extended SSD Lifespan:**: Write operations are costly in terms of lifespan because SSDs have a limited number of write cycles. Disabling journaling could help extend the lifespan of the SSD by reducing the number of write operations, particularly in write-heavy workloads.
+
+
+#### **Consequences of Disabling Journaling**
+
+1. **Increased Risk of Data Corruption**: Without journaling, the system cannot track incomplete or interrupted operations, making the filesystem more vulnerable to corruption in the event of a crash, power failure, or forced reboot. Data in-flight could be lost or corrupted as there is no journal to roll back to a known good state.
+
+2. **Longer Recovery Times**: When journaling is disabled, recovery after a crash or unclean shutdown will take much longer since `fsck` will need to be performed, which is time-consuming on large volumes. Journaling significantly speeds up recovery times by replaying only the necessary changes.
+
+3. **Loss of Filesystem Consistency**: Journaling helps maintain metadata consistency (e.g., file ownership, permissions etc.). Without it, metadata inconsistencies could result in file or directory errors, making files inaccessible or corrupted.
+
+
+
+### Results with Optimizations
+Below are the disk writes in MiB per day after implementing the optimizations proposed above:
+
+| HWSKU | OS Version | Unoptimized | Optimizations A |  Optimizations B | Optimization C |
+| ----- | ---------- | ----------- | --------------- | ---------------- | -------------- |
+| SKU1  | Version_1  | 872.89      | 147.1           | 135.9            | 66             |
+|       | Version_2  | 667.81      | 273.3           | 55.8             | 81             |
+|       | Version_3  | 719.52      | 133.2           | 87.5             | 70.4           |
+| SKU2  | Version_1  | 627.81      | 99.1            | 23.5             | 62.9           |
+|       | Version_3  | 729.14      | 43.3            | 8                | 77             |
+| SKU3  | Version_1  | 762.63      | 237.3           | 175.3            | 50             |
+|       | Version_2  | 591.26      | 160.1           | 94.4             | 52             |
+|       | Version_3  | 643.14      | 96              | 206              | 59             |
+
+##### **Table 4**
+
+Where:
+Optimization A: User space modifications, journaling is enabled and **/tmp is mounted to disk**
```

Comment:
Use bullet points here for readability

Reply:
Thanks for the catch - will add bullet points in upcoming commit.